src/Http/Http.Abstractions/test/PathStringTests.cs

```diff
@@ -49,6 +49,22 @@ public void NotEquals_EmptyPathStringAndNonNullPathString()
             Assert.NotEqual(pathString, PathString.Empty);
         }
 
+        [Fact]
```

Comment:
Change here won't be checked in. Correct?

Reply:
They are already removed.